dicom_standard/extract_ciod_macro_data.py

```diff
@@ -0,0 +1,50 @@
+'''
+Load the CIOD macro tables from DICOM Standard PS3.3, Annex A.
+All CIOD tables are defined in chapter A of the DICOM Standard.
+Output the tables in JSON format, one entry per CIOD.
+'''
+import sys
+import re
+
+from dicom_standard import parse_lib as pl
+from dicom_standard import parse_relations as pr
+from dicom_standard.table_utils import (
+    get_chapter_tables,
+    tables_to_json,
+    get_short_standard_link,
+    get_table_description,
+    table_to_dict,
+)
+
+CHAPTER_ID = 'chapter_A'
+# Include optional "s" at end of "Functional Group" to catch Table A.32.9-2
+TABLE_SUFFIX = re.compile(".*Functional Groups? Macros$")
```

Comment:
I had actually named it to be consistent with `extract_ciod_module_data.py` since the logic is extremely similar: https://github.com/innolitics/dicom-standard/blob/master/dicom_standard/extract_ciod_module_data.py

I can update the docstring, but would it make sense to update both file names then?

Reply:
Hmm, good point RE `extract_ciod_module_data` naming. I honestly think I did a poor job of naming that module and we should probably rename that one too. The word "data" is really just empty!